packages/payment-processor/test/payment/any-to-erc20-batch-proxy.test.ts

```diff
@@ -241,13 +266,14 @@ describe('erc20-batch-conversion-proxy', () => {
             ],
             batchConvVersion,
             wallet,
+            true,
           ),
         ).rejects.toThrowError('the enrichedRequest has no paymentSettings');
       });
       it('should throw an error if the request is ETH', async () => {
         EURRequest.currencyInfo.type = RequestLogicTypes.CURRENCY.ETH;
         await expect(
-          payBatchConversionProxyRequest(enrichedRequests, batchConvVersion, wallet),
+          payBatchConversionProxyRequest(enrichedRequests, batchConvVersion, wallet, true),
```

Comment:
why add true which is the default value ?

Reply:
done thx